plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/service/JavaLanguageExtensionServiceClient.java

```diff
@@ -22,10 +24,15 @@
 import org.eclipse.che.api.promises.client.js.Promises;
 import org.eclipse.che.jdt.ls.extension.api.dto.ExtendedSymbolInformation;
 import org.eclipse.che.jdt.ls.extension.api.dto.FileStructureCommandParameters;
+import org.eclipse.che.jdt.ls.extension.api.dto.GetEffectivePomParameters;
 import org.eclipse.che.plugin.languageserver.ide.service.ServiceUtil;
 
 @Singleton
 public class JavaLanguageExtensionServiceClient {
+
+  private static final String FILE_STRUCTURE = "java/file-structure";
+  private static final String EFFECTIVE_POM = "java/effective-pom";
```

Comment:
I think to have these constants in one place will be better. I suggest move them to `che/plugins/plugin-java/che-plugin-java-ext-lang-shared/src/main/java/org/eclipse/che/ide/ext/java/shared/Constants.java`

Reply:
I am not sure, but probably it should be done in more general way.